pkg/controller/controller.go

```diff
@@ -664,15 +663,23 @@ func (p *csiProvisioner) Delete(volume *v1.PersistentVolume) error {
 	storageClassName := volume.Spec.StorageClassName
 	if len(storageClassName) != 0 {
 		if storageClass, err := p.client.StorageV1().StorageClasses().Get(storageClassName, metav1.GetOptions{}); err == nil {
+
+			// Get PVC for secret reference if the volume is bound.
+			var pvc *v1.PersistentVolumeClaim
```

Comment:
Can you add a unit test for this deletion case?

Reply:
Added simple Delete unit tests.